boards/Makefile.common

```diff
@@ -77,6 +81,9 @@ check_defined = $(strip $(foreach 1,$1,$(if $(value $1),,$(error Undefined varia
 $(call check_defined, PLATFORM)
 $(call check_defined, TARGET)
 
+# Location of target-specific build
+TARGET_PATH = $(TARGET_DIRECTORY)$(TARGET)
```

Comment:
```suggestion
TARGET_PATH := $(TARGET_DIRECTORY)$(TARGET)
```

This is probably a net win to include as the `target/` part of the target path is actually from rustc, not from us, so having it specified in one place makes sense.

Only change is a preference for non-lazy assignment; in this specific instance, effectively a no-op, but as a style / rule, we avoid lazy assignment in make as it can cause hard to track down errors.

Reply:
ok. will check how non-lazy assignment work with consuming TARGET_PATH from board makefile.